src/js/index.js

```diff
@@ -67,6 +65,10 @@ function setupShowcases() {
   const showcase = $('.showcases__project')
   const code = document.getElementById('showcase-code')
   const title = document.getElementById('showcase-title')
+  if (!code) {
```

Comment:
@Serabe `$('.showcases__project')` returns `null` if there is no elements matching the selector